helper/config.go

```diff
@@ -33,8 +33,7 @@ type Config struct {
 	AdminKey               string        `toml:"admin_key"` //used for tools/admin to communicate with yig
 	GcThread               int           `toml:"gc_thread"`
 	LcThread               int           //used for tools/lc only, set worker numbers to do lc
-	LcDebug                bool          //used for tools/lc only, if this was set true, will treat days as seconds
```

Comment:
LcDebug is needed now.

Reply:
I merged this flag with `helper.CONFIG.DebugMode`, cannot imagine a scenario their values are different.